protobuf/service.proto

```diff
@@ -204,7 +204,7 @@ service BESSControl {
   rpc DestroyPort (DestroyPortRequest) returns (EmptyResponse) {}
 
   /// Runtime-updatable configuration
-  rpc SetPortConf (SetPortConfRequest) returns (EmptyResponse) {}
+  rpc SetPortConf(SetPortConfRequest) returns (CommandResponse) {}
```

Comment:
nit: hm, a whitespace was removed.

Reply:
(Interesting, we don't run these through clang-format?)

Reply:
Oh, I see, maybe we don't lint protobuf file. FYI. running clang-format-5.0 gave me this: (Removing a bunch of spaces). I will add back space here.

![image](https://user-images.githubusercontent.com/7221472/73313129-3cb18200-41df-11ea-997b-1b5397ec2428.png)